devcontainer.cht-app-ide/.devcontainer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "image": "cht-ide",
+  "image": "public.ecr.aws/s5s3h4s7/cht-app-ide:latest",
```

Comment:
I _think_ that you can use the `medic` alias here: https://github.com/medic/cht-core/issues/7952